config.md

```diff
@@ -139,6 +139,38 @@ For POSIX platforms the `mounts` structure has the following fields:
 ]
 ```
 
+### <a name="configLinuxMounts" />Linux Mounts
+
+For Linux the `mounts` structure has the following fields:
+
+* **`attr`** (object, OPTIONAL) specifies `mount_setattr(2)` attributes
+    * **`flags`** (array of strings, OPTIONAL) specifies the `flags` argument of `mount_setattr(2)` syscall.
+       e.g.,  "AT_RECURSIVE"
+    * **`set`** (array of strings, OPTIONAL) specifies the `attr_set` field of `mount_attr` struct.
+       e.g., "MOUNT_ATTR_RDONLY"
+    * **`clr`** (array of strings, OPTIONAL) specifies the `attr_clr` field of `mount_attr` struct.
```

Comment:
As all other fields are named exactly as in `mount_setattr(2)`, perhaps we should do the same here?

IOW
* `set` -> `attr_set`
* `clr` -> `attr_clr`

Reply:
Updated, sorry for delay